mysite/announcements/tasks.py

```diff
@@ -40,3 +45,6 @@ def expire_announcements():
         expired_announcement_id.append(announcement.id)
 
     Announcements.objects.filter(id__in=expired_announcement_id).update(has_expired=True)
+    AnnouncementDeliveryStatus.objects.filter(announcement_id__in=expired_announcement_id,
```

Comment:
What will happen if https://github.com/tarungarg546/annoucements_onboarding/pull/10/files#diff-2851f38521826a6b9257588b60a25c5eR47 goes through but this statement fails? 

How do we ensure that either both transaction happens or none?